src/rrdp-squeezed/rrdp_squeezed.ml

```diff
@@ -55,7 +56,7 @@ module MemoryActions = struct
 
 	let watch_token domid = Printf.sprintf "xcp-rrdd-plugins/squeezed:domain-%d" domid
 
-	let watch_fired xc path domains watches =
+	let watch_fired xc _ path domains watches =
```

Comment:
In the PR on `xcp-rrdd`, the signature of this function was `watch_fired _ _xc path domains _`. It seems that `xc` is in different positions: https://github.com/xapi-project/xcp-rrdd/pull/80/files#diff-aae74f219b1c9bcd5b3de6bc44497519R169

Is this a typo in the other PR?

Reply:
The signature is:
```
    val watch_fired : Xenctrl.handle -> Xenstore.Xs.xsh -> string -> Xenctrl.domaininfo IntMap.t -> IntSet.t -> unit
```

So an implementation could use:
```
let watch_fired xc _ path domains watches =
```

I agree that `xc` should come first. However, the first two arguments were not used so this did not introduce a bug.